crates/subspace-core-primitives/src/lib.rs

```diff
@@ -145,6 +190,13 @@ impl AsMut<[u8]> for Randomness {
     }
 }
 
+impl From<Blake3Hash> for Randomness {
+    #[inline]
+    fn from(hash: Blake3Hash) -> Self {
+        Self::from(hash.0)
+    }
+}
```

Comment:
Same question here: are they so common that we need these conversions? I think there are very few specialized places where we may do such conversion and I wouldn't implement a trait for that. Similarly for `PosSeed`.